.ci/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -leE
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
not needed, can use topdir set from settings.sh - less lines -> less code -> less bugs -> less support -> :)

Reply:
SCRIPT_DIR is needed to correctly source settings.sh by absolute paths - verified with main script called from folders on different levels and even when script is called with relative paths.